pkgs/servers/sip/kamailio/default.nix

```diff
@@ -0,0 +1,76 @@
+{ callPackage
+, fetchurl
+, lib
+, pkgs
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kamailio";
+  version = "5.7.1";
+
+  meta = {
+    description = "Fast and flexible SIP server, proxy, SBC, and load balancer";
+    homepage = "https://www.kamailio.org/";
+    license = lib.licenses.gpl2Only;
+    maintainers = with lib.maintainers; [ mawis ];
+    platforms = lib.platforms.linux;
+  };
+
+  src = fetchurl {
+    url = "https://www.kamailio.org/pub/kamailio/5.7.1/src/kamailio-${version}_src.tar.gz";
+    sha256 = "sha256-TkrAj5ohwzJ959UvXNwaMmcIwTRowUitP9/UNQ8D1zw=";
```

Comment:
```suggestion
    hash = "sha256-TkrAj5ohwzJ959UvXNwaMmcIwTRowUitP9/UNQ8D1zw=";
```

Reply:
Thanks, haven't seen that before. I guess I did to much copying from other packages ;)